cloud_infrastructure_access/rosa-aws-privatelink.adoc

```diff
@@ -0,0 +1,17 @@
+[id="rosa-aws-privatelink"]
+= Configuring AWS PrivateLink
+include::modules/attributes-openshift-dedicated.adoc[]
+:context: rosa-aws-privatelink
+
+toc::[]
+
+include::modules/rosa-configure-privatelink-about.adoc[leveloffset=+1]
+include::modules/rosa-configure-privatelink-architecture.adoc[leveloffset=+1]
+include::modules/rosa-configure-privatelink-requirements.adoc[leveloffset=+1]
+include::modules/rosa-configure-privatelink-configuration.adoc[leveloffset=+1]
+
+== Additional resources
+
+* See xref:../rosa_getting_started/rosa-setting-up-environment.adoc[Setting up the environment] to prepare your environment and install {product-title} (ROSA).
+* For an overview of installing ROSA using an existing VPC, see link:https://docs.openshift.com/container-platform/latest/installing/installing_aws/installing-aws-vpc.html#installation-custom-aws-vpc-requirements_installing-aws-vpc[Requirements for using your VPC] in the OCP documentation.
+* If you use a firewall, see link:https://docs.openshift.com/container-platform/latest/installing/install_config/configuring-firewall.html[Configuring your firewall for OpenShift Container Platform] for information about granting access to sites that OCP requires.
```

Comment:
@vikram-redhat Are we able to stop using these "Additional resources" sections yet, and go back to in-line xrefs?? 🙏

Reply:
Also- I have to fix the links- they should go to the portal.

Reply:
access.redhat.com portal? I would actually prefer that they remain docs.openshift.com links

Reply:
I understand, but still have to fix them. (side-stepping out of the line of fire....)

Reply:
@wgordon17 yes the xref fix is coming very soon.